.circleci/config.yml

```
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/node:17
+      - image: circleci/node:20
```

```suggestion
      - image: cimg/node:20.17.0
```

`circleci/node:17` is the highest Node.js version of this deprecated series replaced by `cimg/node`

Docker image `circleci/node:20` does not exist
```text
$ docker pull circleci/node:20
Error response from daemon: manifest for circleci/node:20 not found: manifest unknown: manifest unknown
```

`cimg/node:20` does not exist. Images are now published only with the full Node.js version number. The latest CircleCI Node Docker image with Node.js `20.x` is `cimg/node:20.17.0`.